.gitignore

```diff
@@ -1,25 +1,4 @@
-*.gem
```

Comment:
Some of these still seem useful to keep in gitignore.  IS there a reason for removing?

Reply:
many weren't needed. i made it minimal